fdbserver/storageserver.actor.cpp

```diff
@@ -13421,6 +13732,29 @@ ACTOR Future<bool> restoreDurableState(StorageServer* data, IKeyValueStore* stor
 		}
 	}
 
+	state KeyRangeMap<Optional<UID>> bulkLoadTaskRangeMap; // store dataMoveId on ranges with active bulkload tasks
+	bulkLoadTaskRangeMap.insert(allKeys, Optional<UID>());
+	state RangeResult bulkLoadTasks = fBulkLoadTask.get();
+	for (int i = 0; i < bulkLoadTasks.size() - 1; i++) {
+		if (bulkLoadTasks[i].value.empty()) {
+			continue;
+		}
```

Comment:
Why would this be empty? Do we clear metadata for finished task?

Reply:
The metadata is read from SS local key range space. It is possible that there some holes for some subrange.